blockchain/state/statedb.go

```diff
@@ -890,3 +890,24 @@ func (s *StateDB) GetContractStorageRoot(contractAddr common.Address) (common.Ha
 	}
 	return contract.GetStorageRoot(), nil
 }
+
+// NewStorageTrieIterators returns a list of storage trie iterators from the given
+// storage trie root. Returned iterators are children of storage trie root node.
+func (s *StateDB) NewStorageTrieIterators(storageTrieRoot common.Hash) ([]*statedb.Iterator, error) {
```

Comment:
This function is no longer in use. We could delete this?

Reply:
I thought I've deleted the function. Thanks!